specification/containerservice/Fleet.Management/clusterresourceplacement.tsp

```diff
@@ -0,0 +1,229 @@
+import "@azure-tools/typespec-azure-resource-manager";
+
+using TypeSpec.Versioning;
+
+namespace Microsoft.ContainerService;
+
+@added(Versions.v2025_03_02_naar)
+@doc("The propagation to be used for provisioning the namespace among the fleet.")
+model PropagationPolicy {
+  @doc("The type of the policy to be used. Default is Placement.")
+  type?: PropagationType;
+
+  @doc("The propagation profile to be used for the propagation. Default is PlacementProfile.")
+  propagationProfile?: PropagationProfile;
+}
+
+@added(Versions.v2025_03_02_naar)
+@doc("How the namespace will be provisioned among the fleet members.")
+union PropagationType {
+  @doc("Using ClusterResourcePlacement.")
+  Placement: "Placement",
+
+  string,
+}
+
+@added(Versions.v2025_03_02_naar)
+@doc("The available Propagation profiles.")
+union PropagationProfile {
+  @doc("Use ClusterResourcePlacement.")
+  PlacementProfile,
+}
+
+@added(Versions.v2025_03_02_naar)
+@doc("The configuration profile for default ClusterResourcePlacement for placement.")
+model PlacementProfile {
+  @doc("The default ClusterResourcePlacement policy configuration.")
+  defaultClusterResourcePlacement?: ClusterResourcePlacement;
+}
+
+@added(Versions.v2025_03_02_naar)
+@doc("The desired configuration of ClusterResourcePlacement")
+model ClusterResourcePlacement {
+  @doc("PlacementPolicy defines how to select member clusters to place the selected resources. If unspecified, all the joined member clusters are selected.")
```

Comment:
this docstring does not correspond with the underlying API - please can you re-crosscheck them?

Reply:
I used the spec because initially I thought it was going to be like the spec than the full CRP object.  Corrected this now.